### DISCUSSION TITLE
Question about blank page using wpf.

### DISCUSSION THREAD

**Discussion by krubin723** [cefsharp/CefSharp#3973] [General]
Hi,

1) I registered this morning at EST at 10 AM for the user id krubin723. I did not get an activation email.
2) We have a wpf application:
•	What version of the product are you using?
CefSharp - 75.1.143.0
•	What architecture?
x86
•	On what operating system?
Win 10
•	Are you using WinForms, WPF or OffScreen?
WPF
•	What steps will reproduce the problem?
I have a WPF application in which we are using CEFsharp for displaying web page . There is not html content in the web page. After the screen is minimized and opened again, we can see the web page. One can only use the application if they minimize after every operation. This is only happening on about 10 of our clients out of many hundreds.When we use the debug tools on the blank page that came up, there is code in the view source. 
•	What is the expected output? What do you see instead?
Blank screen comes up. We end up minimizing the application and the page comes up correctly.
 
Thank you,
Ken

**Comment by amaitland** [comment]
> I registered this morning at EST at 10 AM for the user id krubin723. I did not get an activation email.

No idea what you are referring to here.

> This is only happening on about 10 of our clients out of many hundreds

Are they using Intel 11th Gen iris graphics cards? There is a bug in the Intel driver. #3587

**Comment by krubin723** [comment]
I tried to register on this site and did not get an activation email: https://magpcss.org/ceforum/viewforum.php?f=6 -Chromium Embedded Framework Forum
User id krubin723

**Reply by amaitland** [reply]
> I tried to register on this site and did not get an activation email:

See https://bitbucket.org/chromiumembedded/cef/issues/2962/unable-to-register-in-your-forums#comment-58930391

Whilst there is a CefSharp subforum, the forum itself is not run by me.

**Comment by krubin723** [comment]
Hi,

I hope to be able to test out a code solution to my issue above, on Windows 10, in the next couple of days, regarding clients with 11th Gen iris graphics cards and will update this blog with my findings. The ProcessRenderMode will be set to RenderMode.SoftwareOnly;
I have another question however:
On Windows 11, the same application never fully starts up. You see a wheel depicting waiting, but the application just crashes.
• What version of the product are you using?
CefSharp - 75.1.143.0
• What architecture?
x86
• On what operating system?
Win 10
• Are you using WinForms, WPF or OffScreen?
WPF
Windows 11 Pro 10.0.22000 build 22000
Processor Intel Core i3-8100 3.6GHZ, 4 cores, Bios Del Inc 1.14 7/10/2018

Any insights would be greatly appreciated.

Thank you,
Ken

**Reply by amaitland** [reply]
> On Windows 11, the same application never fully starts up. You see a wheel depicting waiting, but the application just crashes.
• What version of the product are you using?
CefSharp - 75.1.143.0

It's entirely possible that version 75 is not compatible with Windows 11. You can check the log file for errors see https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

You can test with version 97 using https://github.com/cefsharp/CefSharp.MinimalExample

**Reply by krubin723** [reply]
Alex,
I will test with the minimal example. What's I forgot to mention is that the application works on some Windows 11 PCs, but not others.

Thank you,
Ken

**Reply by krubin723** [reply]
Hi Alex,

Your fix for windows 10 was very helpful. Thank you! I was finally able to provide it to a client.

I have some more information on my Windows 11 issue, using version 75. The error is from the statement:
parent.Background = new System.Windows.Media.SolidColorBrush(
System.Windows.Media.Color.FromRgb(0xE4, 0xE4, 0xE4)
);
I know this is not from your library. Do you know if it may be incompatible with the use of your library possibly? The error is below: 
 [(null)] - An exception was thrown rendering the UserListCombo box .
System.NullReferenceException: Object reference not set to an instance of an object.

I know this is a stretch!

Thank you,
Ken

**Reply by amaitland** [reply]
I'd be very doubtful thatsetting the background causes CefSharp to break.

Do you have a full stack trace? Cannot say from the exception alone.
